### DISCUSSION TITLE
Welcome to webRTC Discussions!

### DISCUSSION THREAD

**Discussion by haudiobe** [Dash-Industry-Forum/webRTC#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by savan15** [comment]
Hello everyone

i have facing issue in webrtc related stuff

i have integrating webrtc in my android demo app.  i have facing issue is that it is working fine in india but not working in europe countries (call is not established)
main issue is that in europe mobile data network not working. if app is in wifi so it's working 

i have used below turn-stun server 
**stun:stun.l.google.com:19302**

please help me resolve issue
